satellite/gracefulexit/endpoint.go

```diff
@@ -398,43 +412,103 @@ func (endpoint *Endpoint) processIncomplete(ctx context.Context, stream processS
 	return nil
 }
 
-func (endpoint *Endpoint) handleSucceeded(ctx context.Context, pending *pendingMap, nodeID storj.NodeID, message *pb.StorageNodeMessage_Succeeded) (err error) {
+func (endpoint *Endpoint) handleSucceeded(ctx context.Context, pending *pendingMap, exitingNodeID storj.NodeID, message *pb.StorageNodeMessage_Succeeded) (err error) {
 	defer mon.Task()(&ctx)(&err)
-	if message.Succeeded.GetOriginalOrderLimit() == nil {
-		return Error.New("original order limit cannot be nil.")
-	}
-	if message.Succeeded.GetOriginalPieceHash() == nil {
-		return Error.New("original piece hash cannot be nil.")
-	}
 
 	pieceID := message.Succeeded.OriginalPieceId
-	endpoint.log.Debug("transfer succeeded", zap.Stringer("piece ID", pieceID))
-
-	// TODO validation
 
 	transfer, ok := pending.get(pieceID)
 	if !ok {
-		endpoint.log.Debug("could not find transfer message in pending queue. skipping.", zap.Stringer("piece ID", pieceID))
+		endpoint.log.Error("could not find transfer message in pending queue", zap.Stringer("piece ID", pieceID))
```

Comment:
not sure about this - changed it to an error log since I saw the TODO requested an error returned. I guess the log could be taken out now, but I'm not sure how important it is. maybe @ethanadams has an opinion?